bucket/gobang.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "0.1.0-alpha.5",
+    "description": "A cross-platform TUI database management tool written in Rust",
+    "homepage": "https://github.com/TaKO8Ki/gobang",
+    "license": "MIT",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/TaKO8Ki/gobang/releases/download/v0.1.0-alpha.5/gobang-0.1.0-alpha.5-x86_64-pc-windows-msvc.zip",
+            "hash": "4fb7d76776cc778e047b25eb460cdf76745642b937efa07737d0ee077b4541a5"
+        }
+    },
+    "bin": "gobang.exe",
+    "checkver": "github",
```

Comment:
```suggestion
    "checkver": {
        "github": "https://github.com/TaKO8Ki/gobang",
        "regex": "download/v([\\w.-]+)"
    },
```

Reply:
*checkver* only matches `([\\d.]+)` by default. Therefore `"checkver": "github"` does not work.

Reply:
Thank you for your advice.